src/FlysystemOtherManager.php

```diff
@@ -251,7 +251,17 @@ public function __construct($app)
             });
         } elseif (class_exists('\Phlib\Flysystem\Pdo\PdoAdapter')) {
             $this->extend('pdo', function ($app, $config) {
-                return $this->createFlysystem(new \Phlib\Flysystem\Pdo\PdoAdapter(DB::connection($config['database'])->getPdo()));
+                $driverConfig = Arr::only($config, [
+					'database',
+					'visbility',
+					'table_prefix',
+					'enable_compression',
+					'chunk_size',
+					'temp_dir',
+					'disable_mysql_buffering',
```

Comment:
How much of this can/should be configured in the `database.yml`, thereby avoiding the need to support it here?

Reply:
Better question, how much of this can even be overridden here?

Reply:
OK, reading through the adapter's code, I see where these come in. We'll need to pass them in a `Config` object, though, so I'll make a tweak to the PR before I merge it (since there hasn't been any response to the feedback in almost a year).

Reply:
Tweak made.